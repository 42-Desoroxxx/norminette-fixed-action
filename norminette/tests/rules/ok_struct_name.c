typedef struct s_toto t_toto;

int g_int;

struct s_toto g_var;

typedef struct s_toto {
    struct s_toto;
    union u_toto;
    typedef union u_test t_test;
    enum e_toto;
}   t_struct;

t_struct g_var;

int main(void) {
    struct s_type toto;
    return (0);
}