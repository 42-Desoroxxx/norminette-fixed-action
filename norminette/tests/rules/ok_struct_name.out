tests/rules/ok_struct_name.c: OK!